app/assets/javascripts/account/token_copy.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function () {
+  var inputEl = $("#deviceToken");
+  var btnEl = $("#copyBtn");
+
+  if (!inputEl || !btnEl) return;
+
+  inputEl.click(function() {
```

Comment:
that's not a good way to attach event listener

see `$(document).on(...)` for better experience